src/runtime-mpi.c

```diff
@@ -0,0 +1,213 @@
+/* -*- C -*-
+ *
+ * Copyright 2011 Sandia Corporation. Under the terms of Contract
+ * DE-AC04-94AL85000 with Sandia Corporation, the U.S.  Government
+ * retains certain rights in this software.
+ *
+ * Copyright (c) 2018 Intel Corporation. All rights reserved.
+ * This software is available to you under the BSD license.
+ *
+ * This file is part of the Sandia OpenSHMEM software package. For license
+ * information, see the LICENSE file in the top level directory of the
+ * distribution.
+ *
+ */
+
+
+#include "config.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <mpi.h>
+
+#include "runtime.h"
+#include "shmem_internal.h"
+#include "shmem_env.h"
+#include "uthash.h"
+
+#define MAX_KV_COUNT 20 //if more key/values are needed for future features, change this here (should be 2 * the number of keys)
```

Comment:
Please line wrap this comment (you can move it to the line before the #define if you prefer).  Also, we prefer to use old-style `/* ... */` C comments.